compiler/one-cmds/CMakeLists.txt

```diff
@@ -8,6 +10,7 @@ set(ONE_COMMAND_FILES
     one-pack
     one-codegen
     one-prepare-venv
+    one_utils.py
```

Comment:
following `one*` for `one_utils.py` seems confusing for command name similarity. can we use another name that doesn't start with `one` ?